app/src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,27 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { AuthService } from './auth.service';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { JwtPayload } from './interfaces/jwt-payload.interface';
+import { AuthenticationError } from 'apollo-server-core';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+    constructor(private readonly authService: AuthService) {
+        super({
+            jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+            secretOrKey: 'secretKey',
```

Comment:
Move to the `.env` The detailed info here https://docs.nestjs.com/techniques/configuration